### PR TITLE
convert Values to float for influxdb-export

### LINKED ISSUE #16: DataType Problem in Export To influxdb
Some values in inverterDetail Switch at Night From float To int.

Thais causes Type error when exporting To influxdb.

### PR DESCRIPTION
**Pull request by TobiO79**

### CODE REVIEW

**Comment by TobiO79** [comment]
@Gentleman1983 was gefällt pylint hier nicht? kannst du das abstellen?

**Comment by Gentleman1983** [comment]
> @Gentleman1983 was gefällt pylint hier nicht? kannst du das abstellen?

1. you shoud add pylint plugin to PyCharm. It's good to get PyLint results or on-demand scans in your IDE.
2. Issue seems to be a method containing too many statements. you could split that method in multiple sub methods or add a pylint ignore if splitting the method is not practical.
3. it would be great to include the issue id, e.g. `#21` to your commit messages.

File: ginlong_solis_api_connector.py [hunk 1]
```diff
@@ -217,6 +216,83 @@ def get_average_value(inverter_data, field_phase_1, field_phase_2, field_phase_3
             average_value = float((inverter_data[field_phase_1] + inverter_data[field_phase_2] + inverter_data[field_phase_3]) / 3)  # pylint: disable=line-too-long
         return average_value
 
+    def make_dict_detail_float(dict_to_change):
```
Comment (Gentleman1983) [inline ginlong_solis_api_connector.py]: I'd maybe add a second parameter to the method given a link of parameters to convert and iterate over this list of parameters to convert those... Something like:

```
def convert_dict_details_to_float(dict_to_change, parameters):
  for param in parameters:
    dict_to_change[param} = float(dict_to_change[param])

  return dict_to_change
```

and call it like

```
result = convert_dict_details_to_float(my_inverter_dict, ["iPv1", "iPv2", ..., "uAc3"])
```

**Review by Gentleman1983 — CHANGES_REQUESTED** [review]

**Review by Gentleman1983 — APPROVED** [review]
looks good

**Pull request merged**